gammapy/datasets/fermi.py

```diff
@@ -215,6 +217,158 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
+        bin_edges1 =[-(self.x_bins_edges[i] - self.x_cens[i]).value for i in valid_indices]
+        bin_edges2 = [(self.x_bins_edges[i+1] - self.x_cens[i]).value for i in valid_indices]
+
+        y_vals = [y_vals[i] / x_vals[i] for i in range(0, np.size(y_vals))]
+        y_upper = [y_upper[i] / x_vals[i] for i in range(0, np.size(y_vals))]
+        y_lower = [y_lower[i] / x_vals[i] for i in range(0, np.size(y_vals))]
+
+        y_cens = np.array([gmean([y_lower[i], y_upper[i]]) for i in range(0, np.size(y_lower))])
+
+        y_upper = np.array([y_upper[i] - y_vals[i] for i in range(0, np.size(y_lower))])
+        y_lower = np.array([y_vals[i] - y_lower[i] for i in range(0, np.size(y_lower))])
+
+        ax.loglog()
+
+        ax.errorbar(x_vals, y_vals,
+                    yerr=(y_lower, y_upper),
+                    elinewidth=1, linewidth=0, color='black')
+
+        # Place the x-axis uncertainties in the center of the y-axis uncertainties.
+        ax.errorbar(x_vals, y_cens,
+                    xerr=(bin_edges1, bin_edges2),
+                    elinewidth=1, linewidth=0, color='black')
+
+        x_model = np.logspace(np.log10(min(x_vals)), np.log10(max(x_vals)), 25)
+
+        if self.spec_type == "PowerLaw":
+
+            y_model = PowerLaw1D(amplitude=self.int_flux,
+                                 x_0=self.pivot_en,
+                                 alpha=self.spec_index)
+
+        elif self.spec_type == "LogParabola":
+
+            y_model = LogParabola1D(amplitude=self.int_flux,
+                                    x_0=self.pivot_en,
+                                    alpha=self.spec_index,
+                                    beta = self.beta)
+
+        elif self.spec_type == "PLExpCutoff":
+
+            y_model = ExponentialCutoffPowerLaw1D(amplitude=self.int_flux,
+                                                  x_0=self.pivot_en,
+                                                  alpha=self.spec_index,
+                                                  x_cutoff=self.cutoff)
+        elif self.spec_type == "PLSuperExpCutoff":
+            raise NotImplementedError
+
+        ax.set_xlabel('Energy (MeV)')
+        ax.set_ylabel('Flux (ph/cm^2/s/MeV)')
+        ax.plot(x_model, y_model(x_model))
+
+        return ax
+
+    def info(self):
+        """Print the object name, position, flux, and detection signifiance."""
+        info_array = []
+        info_array.append(" ")
+        info_array.append(self.name_3FGL)
+        info_array.append(" ")
+        info_array.append("RA (J2000) " + str(self.ra))
+        info_array.append("Dec (J2000) " + str(self.dec))
+        info_array.append("l " + str(self.glon))
+        info_array.append("b " + str(self.glat))
+        info_array.append("Flux " + str(self.int_flux) + " +/- " + str(self.unc_int_flux)
+                          + " ph /cm2 /MeV /s")
+        info_array.append("Detection significance: " + str(self.signif) + " sigma")
+
+        return info_array
```

Comment:
At the moment all other info columns return a string, so can you do `'\n'.join(info_array)` here or just build a string from the start?
```
ss = 'some info\n'
ss += 'more info\n'
...
return ss
```